### PR TITLE
prmon: add v3.1.0

### PR DESCRIPTION
**Pull request by wdconinc**
This adds a new minor release of prmon, which brings with it nvidia GPU monitoring. No build system changes.

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Verified Checksums. Looks good to me. Waiting on final approval from @vvolkl, @graeme-a-stewart, @amete.

**Review by vvolkl — APPROVED** [review]

**Pull request merged**
